code/go/0chain.net/smartcontract/storagesc/stakepool.go

```diff
@@ -345,8 +347,9 @@ func (ssc *StorageSmartContract) getOrUpdateStakePool(
 }
 
 type stakePoolRequest struct {
-	BlobberID string `json:"blobber_id,omitempty"`
-	PoolID    string `json:"pool_id,omitempty"`
+	BlobberID  string        `json:"blobber_id,omitempty"`
+	PoolID     string        `json:"pool_id,omitempty"`
+	LockPeriod time.Duration `json:"lock_period"`
```

Comment:
The lock period seems not necessary as we will only check if the we are able to unlock a pool by comparing the time.Now() to the stakedAt + minLockPeriod.

Reply:
fixed